students/rmart300/lesson02/fizz_buzz.py

```diff
@@ -0,0 +1,14 @@
+
+
+for i in range(100):
```

Comment:
you can specify start and end for your range like:
`for i in range(1, 101)`

this way you don't need to `i + 1`